README.md

```diff
@@ -44,6 +46,24 @@
 
 The example app is a catalogue of the core components provided by this library. With little effort, you can see the functionality of each component using your real data. You can also modify the example code to see how you can customize Discovery Components to fit your needs.
 
+### Setup script
```

Comment:
Small nitpick: can you add the command to run the script, and maybe specify which directory to run it from?
```
./examples/discovery-search-app/scripts/runExampleApp.sh
```
or, if it's supposed to be run in the example app dir, this:
```
cd examples/discovery-search-app && ./scripts/runExampleApp.sh